src/Altinn.AccessManagement.Core/Helpers/DelegationHelper.cs

```diff
@@ -74,6 +74,38 @@ public static bool TryGetUserIdFromAttributeMatch(List<AttributeMatch> match, ou
             return false;
         }
 
+        /// <summary>
+        /// Trys to get the organization number attribute value from a list of AttributeMatch models
+        /// </summary>
+        /// <returns>The organization number if found as the single attribute in the collection</returns>
+        public static bool TryGetOrganizationNumberFromAttributeMatch(List<AttributeMatch> match, out string orgNo)
+        {
+            orgNo = string.Empty;
+            if (match?.Count == 1 && match.First().Id == AltinnXacmlConstants.MatchAttributeIdentifiers.OrganizationNumberAttribute)
+            {
+                orgNo = match.First().Value;
+                return true;
+            }
+
+            return false;
+        }
+
+        /// <summary>
+        /// Trys to get the social security number attribute value from a list of AttributeMatch models
+        /// </summary>
+        /// <returns>The social security number if found as the single attribute in the collection</returns>
+        public static bool TryGetSocialSecurityNumberAttributeMatch(List<AttributeMatch> match, out string ssn)
+        {
+            ssn = string.Empty;
+            if (match?.Count == 1 && match.First().Id == AltinnXacmlConstants.MatchAttributeIdentifiers.SocialSecurityNumberAttribute)
```

Comment:
Why do the match count need to be 1? Is it already filtered based on socialnumberattriobute when using this method? If so, why do check this at all

Reply:
I realised this as well during implementation. The cause is that the input models used have used a List<AttributeMatch>.
The question I guess is: will we ever need a list of attributes in order to identify a subject or reportee? Or will it always be just a single userid/partyId/ssn/org.no attribute?

For input I guess not? But for the actual rules in the delegation policy there is a potential for additional attributes to be included in the delegated rules and policy.